packages/react-components/src/InputNumber.tsx

```diff
@@ -240,7 +243,9 @@ export default function InputNumber (props: Props): React.ReactElement<Props> {
 
     if (event.key.length === 1 && !isPreKeyDown) {
       const { selectionStart: i, selectionEnd: j, value } = event.target as HTMLInputElement;
-      const newValue = `${value.substring(0, i || 0)}${event.key}${value.substring(j || 0)}`;
+      let newValue = `${value.substring(0, i || 0)}${event.key}${value.substring(j || 0)}`;
```

Comment:
As we agreed on commenting practice  https://github.com/cennznet/apps/pull/3/files#diff-0512f381d37e2f1abc301259b7f7abb9R19 , its better to add `// Commented for easy tracking while rebasing'` and comment the original code? Otherwise its hardly for us to know which piece of code we should keep when polkadot updates their code~

Reply:
Would you mind adding easy tracking comment for this PR as well - https://github.com/cennznet/apps/pull/8/files ? 
If we agreed this approach is too waste of time, we could have a discussion and remove this rule from our PR practice, otherwise, it seems more easier for 'polkadot/apps' long-term maintaining, what do you think?